README.md

```diff
@@ -12,10 +12,7 @@ Column Name | Description
 story_id | The identifier for the story. Comes from the member name in the CNN stories package.
 story_text | The text for the story.
 question | A question about the story.
-answer_char_ranges | Character based indices to answers in story_text. E.g. `196:228|196:202,217:228|None`. Answers from different crowdsourcers are separated by `|`, within those, multiple selections from the same crowdsourcer are separated by `,`.  `None` means the crowdsourcer thought there was no answer to the question in the story. The start is inclusive and the end is exclusive. The end may point to whitespace after a token.
-is_answer_absent | Proportion of crowdsourcers that thought there was no answer to the question in the story.
-is_question_bad | Proportion of crowdsourcers that thought the question does not make sense.
-validated_answers | After crowdsourcing, we validated some answers when consensus was required. This shows how crowdsourcers voted during validation. E.g. `{"none": 1, "294:297": 2}` means that 1 crowdsourcer thought that none of the answers were good and 2 crowdsourcers thought that `294:297` was the best answer.
+answer_token_ranges | word based indices to answers in story_text. E.g. `196:202,217:228`. Multiple selections from the same answer are separated by `,`. The start is inclusive and the end is exclusive. The end may point to whitespace after a token.
```

Comment:
We should still keep these char based indices.  The tokenized version of the dataset is optional.